### ISSUE TITLE
SoftTarget Regularization - feature request - alternative to Dropout (vanilla)

### ISSUE THREAD

**Issue by easyrider** [fchollet/keras#3858]
It's probably a better way to prevent overfitting
As described here:

http://arxiv.org/abs/1609.06693

**Comment by fchollet** [comment]
The paper mention that the research was done using Keras, so the quickest way for you to use SoftTarget would be to ask the author for his code.

**Comment by ArmenAg** [comment]
@fchollet Author here. The code for this paper is a method that wraps model.predict and model.fit. It does not behave like a standard layer (Dropout) or regularizer (L1). What is your preferred approach in this case?
